guides/common/modules/proc_updating-disconnected-server-on-EL8.adoc

```diff
@@ -0,0 +1,148 @@
+[id="Updating-Disconnected-{project-context}-on-EL8_{context}"]
+= Updating a disconnected {ProjectServer} on {EL} 8
+
+.Procedure on the connected {ProjectServer}
```

Comment:
I suggest to add a short introduction to the procedure about the general workflow. the heading and subheading to short succession are slightly confusing to me.